app/src/main/res/drawable/card.xml

```diff
@@ -4,10 +4,10 @@
     <item>
         <shape>
             <padding
-                android:top="8dp"
-                android:right="8dp"
-                android:bottom="5dp"
-                android:left="8dp"/>
+                android:top="0dp"
```

Comment:
8dp left and right padding was added to align widget with widgets like google now.
![](http://i.imgur.com/JZ4bEzf.png)
Bottom padding is needed to show card shadow. Top padding is useless.